ori/volume/server/volume.go

```diff
@@ -16,235 +16,143 @@ package server
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
 	"strings"
+	"time"
 
+	"github.com/go-logr/logr"
 	"github.com/google/uuid"
-	"github.com/onmetal/cephlet/ori/volume/apiutils"
-	"github.com/onmetal/cephlet/pkg/ceph"
-	storagev1alpha1 "github.com/onmetal/onmetal-api/api/storage/v1alpha1"
+	orimeta "github.com/onmetal/onmetal-api/ori/apis/meta/v1alpha1"
 	ori "github.com/onmetal/onmetal-api/ori/apis/volume/v1alpha1"
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/types"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	userIDKey  = "userID"
-	userKeyKey = "userKey"
-
-	pvPoolKey               = "pool"
-	monitorConfigMapDataKey = "csi-cluster-config-json"
-	pvImageNameKey          = "imageName"
-	pvRadosNamespaceKey     = "radosNamespace"
+	// worldwide number key
+	// to use WWN Company Identifiers, set wwnPrefix to Private "1100AA"
+	wwnPrefix string = ""
```

Comment:
Why this constant if it's `""` by default? If it should be overridable, it should be a `var` / a config field of an entity that uses it.